local/assets/ignition/etcd-proxy.yml

```diff
@@ -11,12 +11,13 @@ systemd:
             Environment="ETCD_LISTEN_CLIENT_URLS=http://0.0.0.0:2379"
             Environment="ETCD_INITIAL_CLUSTER={{ range $i, $node := .env.metadata.etcd_ignition_nodes }}{{ if $i }},{{end}}{{$node.name}}=http://{{$node.ip}}:2380{{end}}"
 
-{{ if index .env "ssh_authorized_keys" }}
 passwd:
   users:
     - name: core
+      password_hash: {{ .boot.metadata.root_pass }}
```

Comment:
we only have one machine for demos that is booted as a proxy machine, lets make this the example for `host.metadata.ssh_authorized_keys`